<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWaypointDialog</class>
 <widget class="QDialog" name="UserWaypointDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Little Navmap - Edit Position</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="labelRouteUserWaypointLatLon">
     <property name="text">
      <string>&amp;Latitude and Longitude:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditRouteUserWaypointLatLon</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditRouteUserWaypoint"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditRouteUserWaypointLatLon">
     <property name="toolTip">
      <string>Recognized coordinate formats:
- Degrees, minutes and seconds : N49° 26' 41.57&quot; E9° 12' 5.49&quot; or 49° 26' 41.57&quot; N 9° 12' 5.49&quot; E
- Degrees and minutes: N54* 16.82' W008* 35.95', N 52 33.58 E 13 17.26 or 49° 26.69' N 9° 12.09' E
- Degrees only: 49.4449° N 9.2015° E or N 49.4449° E 9:2015°
Degree, minute and second signs can be omitted for above formats.
Latitude has to be first and N/S and E/W designators are required.
A space is only needed to separate latitude and longitude or degrees/minutes/seconds if no signs used.
Other formats:
- Degrees and minutes: N44124W122451, N14544W017479 or S31240E136502
- Degrees only 46N078W
- Degrees and minutes 4620N07805W
- Degrees, minutes and seconds 481200N0112842E (Skyvector)
- Degrees and minutes in pair &quot;N6500 W08000&quot; or &quot;N6500/W08000&quot;
- NAT type 5020N
Formats above have to be given exactly as shown in the examples.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelRouteUserWaypoint">
     <property name="text">
      <string>&amp;Name of Position:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditRouteUserWaypoint</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="labelRouteUserWaypointCoordStatus">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBoxRouteUserWaypoint">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditRouteUserWaypoint</tabstop>
  <tabstop>lineEditRouteUserWaypointLatLon</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBoxRouteUserWaypoint</sender>
   <signal>accepted()</signal>
   <receiver>UserWaypointDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBoxRouteUserWaypoint</sender>
   <signal>rejected()</signal>
   <receiver>UserWaypointDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
